source/docs/1.0/usage/Notification-Routing.md

```diff
@@ -1,15 +1,57 @@
 ## Notification routing
 
-This is how events are turned into notifications, and how notifications are routed to contacts:
+The graphic below shows how events are turned into
+notifications, and how notifications are routed to contacts as alerts. Further down, you
+can find a step-by-step description of the process.
 
 ![notification routing](/images/notification-routing.gif)
 
-#### Creating the above animation
+## Details
 
-1. Export [architecture diagrams](/images/FlapjackArchitecture.key) to PNG
-2. Run this shell:
+1. **Receive events.**
+    * Events are generated by external check execution systems and created as json objects in the `events` queue in Redis.
+    * Processor removes events off the queue and, astonishingly, processes them one by one.
```

Comment:
The tense on this line doesn't match the rest of the document, and astonishingly probably isn't needed.